core/templates/dev/head/css/oppia.css

```diff
@@ -1027,6 +1030,17 @@ textarea {
   min-width: 126px;
 }
 
+.oppia-library-group-header-active {
```

Comment:
Ok that means that additive class just changes color (to avoid duplication of other css properties). Gotcha.